platform/security/rest/security-rest-cxfwrapper/src/test/java/org/codice/ddf/cxf/paos/PaosInInterceptorTest.java

```diff
@@ -115,6 +121,7 @@ HttpResponseWrapper getHttpResponse(
     assertThat(IOUtils.toString(message.getContent(InputStream.class)), is("actual content"));
     Map<String, List<String>> headers = (Map) message.get(Message.PROTOCOL_HEADERS);
     assertThat(headers.get(testHeaderKey), hasItem(correctHeaderToBeForwarded));
+    assertThat(headers.get(listOfIntsHeaderKey), hasItems("1", "2", "3"));
```

Comment:
ℹ️ Double checking here that something coming in as a list of Integers (as the header was created above) is now a list of strings.